.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
There are a bunch of other files in our current `.gitignore` should we migrate those over as well?

Reply:
The other `.gitignore` files still apply and I don't anticipate many of those files will appear at the root. We can unify at some point but no need right now